actors/evm/src/lib.rs

```diff
@@ -51,12 +51,14 @@ fn test_method_selector() {
 #[repr(u64)]
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
-    // TODO: Do we want to use ExportedNums for all of these, per FRC-42?
     InvokeContract = 2,
```

Comment:
@arajasek, do we have a convention for methods that haven't been deployed yet? Should we _just_ have the exported method and drop the internal one (given that it's not used in this case, anyways?)

Reply:
Yes -- ideally we would _only_ have the exported methods, we just can't do that for methods that have already been deployed. In this case, dropping the internal ones is the right thing to do.

Note that this will still break existing code in Lotus (and go-state-types). I still think it's the right thing to do, but please do the integration work if possible (or at least open issues describing this change) as part of landing this PR.

Reply:
Will do. I'll open a checklist as we have a whole list of changes we need to bubble up at this point.